sample-apps/segmentation/main.py

```diff
@@ -59,9 +59,7 @@ def __init__(self, app_dir, studies, conf):
     def init_infers(self):
         infers = {
             "segmentation": MyInfer([self.pretrained_model, self.final_model], self.network),
-            "Coldstart->ISeg+GraphCut": GenericISegGraphcutColdstart(),
-            "ISeg+GraphCut": GenericISegGraphCut(),
-            "ISeg+SimpleCRF": GenericISegSimpleCRF(),
+            "ModelFreeGraphCut": GenericISegGraphcutModelFree(),
```

Comment:
lets rename this to **FreeGraphCut**

Reply:
What about HistogramBasedGraphCut? There is always a model, in this case a histogram if I'm not mistaken...

Reply:
I like HIstogramBasedGraphCut, will update the apps. I think here by model-free we meant it does not need output from an external model (as it builds its own during execution).